cluster/get-kube-local.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will download latest version of kubectl command line tool and will
+# bring up a local Kubernetes cluster with a single node.
+#
+# Usage:
+#   wget -q -O - https://get.k8s.io/local | bash
+# or
+#   curl -sS https://get.k8s.io/local | bash
+#
+# On Mac OS X you'll have to additionally pass env variable:
+#   wget -q -O - https://get.k8s.io/local | KUBE_HOST=<docker_machine_ip> bash -c
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_HOST=${KUBE_HOST:-localhost}
+
+RED="\033[0;31m"
+GREEN="\033[0;32m"
+ORANGE="\033[0;33m"
+
+function echo_green {
+  echo -e "${GREEN}$1"; tput sgr0
```

Comment:
I think it's equivalent, however if you prefer I can change that if you prefer it.